web/src/main/kotlin/App.kt

```diff
@@ -1,20 +1,22 @@
 import com.surrus.common.remote.Assignment
-import com.surrus.common.remote.PeopleInSpaceApi
 import react.*
 import react.dom.*
 import kotlinx.coroutines.*
 
 
+
 val App = functionalComponent<RProps> { _ ->
-    val scope = MainScope()
-    val api = PeopleInSpaceApi()
+    val mainScope = MainScope()
```

Comment:
In my projects I moved the mainScope declaration either into the body of `useEffectWithCleanup` or used 
`val mainScope = useMemo({ MainScope() }, arrayOf())`. In JetPack compose the [same concept exists right here](https://developer.android.com/reference/kotlin/androidx/compose/package-summary#remember)